content/runbooks/rds/RDSCPUUtilization.md

```diff
@@ -0,0 +1,39 @@
+---
+title: CPU utilization
```

Comment:
I removed the component name from the title of the page in the menu to avoid redundant information and have light interface to navigate.

The exact name of the alert is still the title of the page on the document and on the URL (used by the runbook)